README.md

```diff
@@ -5,12 +5,16 @@ rules that encourage consistency, safety and readability across your code base.
 
 Read [this](https://timjames.dev/blog/the-best-eslint-rules-for-react-projects-30i8) blog post describing my general approach.
 
+Note that ESLint is in the midst of a major upgrade to version 9, which brings with it significant changes to the configuration syntax. The AirBnB configuration - among other related tools - does not yet support v9. For now, all our documentation and approach refers to the 'legacy' ESLint syntax. Therefore, when using this config, make sure to stick with: 
+* ESLint 8
+* TSESLint 7 (/parser and /eslint-plugin)
+
 ## Install
 
 ### `npm`
 
 ```sh
-npm install --dev @tim-w-james/eslint-config \
+npm install -D @tim-w-james/eslint-config \
   @typescript-eslint/eslint-plugin \
   @typescript-eslint/parser \
   eslint \
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought (non-blocking):** ‏We may want to pin these deps from the install command (like `eslint@^8.0.0`), since they'd install the latest by default.

I'll consider making that change in the future if it isn't made in this PR.

Reply:
Done.